UnityProject/Assets/Tests/Utils.cs

```diff
@@ -109,5 +110,36 @@ public static IEnumerable<string> GUIDsToPaths(IEnumerable<string> guids, Predic
 		/// be accessed. If the instance is a true null, then 0 is returned.
 		/// </summary>
 		public static int GetInstanceID(Object instance) => instance is null ? 0 : instance.GetInstanceID();
+
+		private const string MANAGER_PATH = "Assets/Prefabs/SceneConstruction/NestedManagers";
+
+		/// <summary>
+		/// Get singleton manager prefab
+		/// </summary>
+		public static T GetManager<T>(string managerName) where T : SingletonManager<T>
```

Comment:
I think it was Copern who added a shared assembly between the game and editor scripts. Could we do a similar thing w/ tests?

Reply:
Should be able to if SingletonManager is also moved over to the shared assembly.